resources/js/tryitout.js

```diff
@@ -51,6 +51,17 @@ function cancelTryOut(endpointId) {
     }
 }
 
+function recursiveAddSearchParams(url, previousAppend, previousKey, obj) {
```

Comment:
Also no need to pass in the full `url`. `url.searchParams` is what we need, so you can change the parameter to `searchParams`.